.flaskenv

```diff
@@ -2,5 +2,4 @@ FLASK_APP=app.py
 FLASK_ENV=development
 FLASK_RUN_PORT=5000
 FLASK_RUN_HOST=localhost
-DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/application_store
-USE_LOCAL_DATA=True
+DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:7890/fsd_app_store_test
```

Comment:
Should the port remain the default for postgres 5432?

Reply:
I've used what's on main (7890) but 5432 would make more sense to me as that's how it will run our machines and the docker runner? can change this back to 5432 if we're happy with that? 👍🏾